### PR TITLE
API : Epoch | epoch scenario

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
image

**Comment by ghost** [comment]
image

File: src/main/java/util/RamdomNumber.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package util;
+import java.util.Random;
+
+public class RamdomNumber {
+    public int ramdomInger(int from){
```
Comment (Sotatek-DavidNguyen4) [inline src/main/java/util/RamdomNumber.java]: randomInteger

File: src/main/java/microservices/epoch/steps/EpochSteps.java [hunk 2]
```diff
@@ -213,6 +218,29 @@ public EpochSteps then_verifyFormatOfBlockListResponse(List<EpochDataByEpochNo>
...
+        assertThat(epochDetailInListEpoch.getOutSum()).isEqualTo(epochDetail.getOutSum());
+        assertThat(epochDetailInListEpoch.getTxCount()).isEqualTo(epochDetail.getTxCount());
+        assertThat(epochDetailInListEpoch.getStartTime()).isEqualTo(epochDetail.getStartTime());
+        assertThat(epochDetailInListEpoch.getEndTime()).isEqualTo(epochDetail.getEndTime());
+        assertThat(epochDetailInListEpoch.getMaxSlot()).isEqualTo(epochDetail.getMaxSlot());
+        assertThat(epochDetailInListEpoch.getRewardsDistributed()).isEqualTo(epochDetail.getRewardsDistributed());
+        return this;
+    }
+    @Step("compare api get all epoch with api get epoch current")
+    public EpochSteps compareAllEpochWithCurrentEpoch(Epoch epoch, EpochCurrent epochCurrent){
```
Comment (Sotatek-DavidNguyen4) [inline src/main/java/microservices/epoch/steps/EpochSteps.java]: compare all but i don't see all. You are getting just one?

**Review by Sotatek-GiangNguyen7 — APPROVED** [review]

**Pull request merged**